connection.go

```diff
@@ -433,13 +432,13 @@ func (c *conn) handleStagingPut(ctx context.Context, presignedUrl string, header
 	}
 	client := &http.Client{}
 
-	dat, err := os.ReadFile(localFile)
+	dat, err := os.Open(localFile)
 
 	if err != nil {
 		return dbsqlerrint.NewDriverError(ctx, "error reading local file", err)
 	}
 
-	req, _ := http.NewRequest("PUT", presignedUrl, bytes.NewReader(dat))
+	req, _ := http.NewRequest("PUT", presignedUrl, dat)
```

Comment:
Should the file be closed after sending a request? Sorry if this question is stupid, but I'm not very familiar with Go 🤷‍♂️

Reply:
@kravets-levko yes! good point, added `defer dat.Close()`